src/Notes/WC_Admin_Notes_Giving_Feedback_Notes.php

```diff
@@ -0,0 +1,65 @@
+<?php
+/**
+ * WooCommerce Admin (Dashboard) Giving feedback notes provider
+ *
+ * Adds notes to the merchant's inbox about giving feedback.
+ *
+ * @package WooCommerce Admin
+ */
+
+namespace Automattic\WooCommerce\Admin\Notes;
+
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * WC_Admin_Notes_Giving_Feedback_Notes
+ */
+class WC_Admin_Notes_Giving_Feedback_Notes {
+	/**
+	 * Note traits.
+	 */
+	use NoteTraits;
+
+	/**
+	 * Add notes for admin giving feedback.
+	 */
+	public static function add_notes_for_admin_giving_feedback() {
+		self::possibly_add_admin_giving_feedback_note();
+	}
+
+	/**
+	 * Possibly add a notice setting moved note.
+	 */
+	protected static function possibly_add_admin_giving_feedback_note() {
+		$name       = 'wc-admin-store-notice-giving-feedback';
```

Comment:
I think we'll want to update this to a new name.  If a user had the previous `wc-admin-store-notice-giving-feedback` note and skipped a version of wc-admin where it was removed to this version, the note would not be updated to the new content.

Reply:
Great point, I've made it unique now.